state/state.go

```diff
@@ -7,9 +7,16 @@ import (
 	"context"
 
 	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/set"
 	"github.com/ava-labs/avalanchego/x/merkledb"
 )
 
+const (
+	Read          uint8 = 0
```

Comment:
I didn't realize `bit.Set` used `int`...for that reason, it is probably ok to keep that and then just ensure it is a valid enum below.